oauth2client/contrib/locked_file.py

```diff
@@ -122,8 +122,8 @@ def open_and_lock(self, timeout, delay):
             CredentialsFileSymbolicLinkError if the file is a symbolic link.
         """
         if self._locked:
-            raise AlreadyLockedException('File %s is already locked' %
-                                         self._filename)
+            raise AlreadyLockedException('File {0} is already locked'.format(
```

Comment:
The principle of always-line-wrap-at-the-coarsest-level-of-syntax applies here. By wrapping after the opening parenthesis you still have the `AlreadyLockedException`-typed expression broken across lines, but you can put on a single line the subexpression that is the argument of the `AlreadyLockedException` construction.